### PR TITLE
Add a max-width to user name in the header, fixes #7676

### LINKED ISSUE #7676: Navbar too high on small screens in combination with long username
When the username is long and the width of the screen is small the navbar is higher than normal. This wouldn't be much of a problem if it wasn't blocking the view of the main page. When composing a message the buttons on top (preview and others) are not visible.

screenshot

### PR DESCRIPTION
**Pull request by Flaburgan**
300px is small enough, if the screen width get small the header becomes responsive so the user name is not displayed anymore.

### CODE REVIEW

File: app/assets/stylesheets/header.scss [hunk 1]
```diff
@@ -119,6 +119,19 @@
     }
   }
 
+  .dropdown-toggle {
+    align-items: center;
+    display: flex;
+
+    .user-name {
+      display: inline-block;
```
Comment (SuperTux88) [inline app/assets/stylesheets/header.scss:127]: I think this isn't needed, at least my browser (firefox and chrome) is telling me, that it's using `display: block` anyway and just ignores it.

**Review by SuperTux88 — CHANGES_REQUESTED** [review]
Looks good otherwise

**Comment by SuperTux88** [comment]
Oh, I also think 300px is too much, it works with English, but still breaks with some other languages (German for example). So maybe just reduce it to 250px so we have some more space for languages with longer texts in the header?

**Comment by Flaburgan** [comment]
Done. I also added a small margin between the name and the caret.

**Review by SuperTux88 — APPROVED** [review]

**Comment by SuperTux88** [comment]
Merged as 37f081959c8ea1aad7bd23481a34d8ca5815bb13

Thanks for the fix :cookie:

**Pull request closed without merging**